dashboard/db/migrate/20230503220558_create_lti_integrations.rb

```diff
@@ -0,0 +1,20 @@
+class CreateLtiIntegrations < ActiveRecord::Migration[6.0]
+  def change
+    create_table :lti_integrations do |t|
```

Comment:
Look into using [validations](https://guides.rubyonrails.org/active_record_validations.html) for some of these fields that should be required.

Reply:
Required:
* platform_id
* issuer
* client_id
* platform_name
* auth_redirect_url
* jwks_url
* access_token_url